srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php

```diff
@@ -505,15 +505,15 @@ public function getPvpRewardsRender($amount, $isWinner, $bracket, $month, $year,
                                             </th>
                                             <td>
                                                 <select name="bracket" id="bracket" required>
-                                                    <option value=0 <?php if ($bracket == 0) echo 'selected'; ?>>All</option>
-                                                    <option value=1 <?php if ($bracket == 1) echo 'selected'; ?>>10-19</option>
-                                                    <option value=2 <?php if ($bracket == 2) echo 'selected'; ?>>20-29</option>
-                                                    <option value=3 <?php if ($bracket == 3) echo 'selected'; ?>>30-39</option>
-                                                    <option value=4 <?php if ($bracket == 4) echo 'selected'; ?>>40-49</option>
-                                                    <option value=5 <?php if ($bracket == 5) echo 'selected'; ?>>50-59</option>
-                                                    <option value=6 <?php if ($bracket == 6) echo 'selected'; ?>>60-69</option>
-                                                    <option value=7 <?php if ($bracket == 7) echo 'selected'; ?>>70-79</option>
-                                                    <option value=8 <?php if ($bracket == 8) echo 'selected'; ?>>80</option>
+                                                    <option value=0 <?php if ($data['bracket'] == 0) echo 'selected'; ?>>All</option>
+                                                    <option value=1 <?php if ($data['bracket'] == 1) echo 'selected'; ?>>10-19</option>
+                                                    <option value=2 <?php if ($data['bracket'] == 2) echo 'selected'; ?>>20-29</option>
+                                                    <option value=3 <?php if ($data['bracket'] == 3) echo 'selected'; ?>>30-39</option>
+                                                    <option value=4 <?php if ($data['bracket'] == 4) echo 'selected'; ?>>40-49</option>
+                                                    <option value=5 <?php if ($data['bracket'] == 5) echo 'selected'; ?>>50-59</option>
+                                                    <option value=6 <?php if ($data['bracket'] == 6) echo 'selected'; ?>>60-69</option>
+                                                    <option value=7 <?php if ($data['bracket'] == 7) echo 'selected'; ?>>70-79</option>
+                                                    <option value=8 <?php if ($data['bracket'] == 8) echo 'selected'; ?>>80</option>

```

Comment:
:S, I suggest to use a for cycle here, like:
```php
<?php
foreach (brackets as $key, bracket) {
?>
<option value="<?= $key ?>" <?php if $data['bracket'] == $key) { echo 'selected'; } ?> ....bracket... </option>
<?php
}
?>
```

something similar, if you have no time, let's refactor this later

Reply:
Done!